packages/strapi-parts/src/MainNav/MainNav.stories.mdx

```diff
@@ -78,4 +78,50 @@ Description...
       );
     }}
   </Story>
+  <Story name="notifications">
+    {() => {
+      const [condensed, setCondensed] = useState(false);
+      return (
+        <Box background="neutral100" padding={10} style={{ height: '100vh' }}>
+          <MainNav condensed={condensed}>
+            <NavBrand workplace="Workplace" title="Strapi Dashboard" icon={<img src={strapiImage} alt="" />} />
+            <Divider />
+            <NavSections>
+              <NavLink to="/cm" icon={<ContentIcon />} className="active">
+                Content
+              </NavLink>
+              <NavSection label="Plugins">
+                <NavLink to="/builder" icon={<CtbIcon />}>
+                  Builder
+                </NavLink>
+                <NavLink to="/content" icon={<MediaLibIcon />}>
+                  Media library
+                </NavLink>
+                <NavLink to="/content" icon={<AlertInfoIcon />}>
+                  Documentation
+                </NavLink>
+              </NavSection>
+              <NavSection label="General">
+                <NavLink to="/builder" icon={<PluginsIcon />}>
+                  Plugins
+                </NavLink>
+                <NavLink to="/content" badgeContent={'1'} icon={<MarketPlaceIcon />} badgeAriaLabel="notifications">
```

Comment:
In my opinion, for the notification case, the badgeAriaLabel should be
`badgeAriaLabel={${numberOfNotification} notification(s).}`

Also, the badgeContent for the notification case should be a number `badgeContent={1}`